ci/build_libtelio.py

```diff
@@ -317,7 +329,49 @@ def main() -> None:
         assert False, f"command '{args.command}' not supported"
 
 
+def exec_bindings(args):
+    if args.try_fetch_from_pipeline:
+        if "LLH_GROUP_TOKEN_FLAKY_TESTS" in os.environ:
+            token = os.environ["LLH_GROUP_TOKEN_FLAKY_TESTS"]
+        else:
+            token = getpass.getpass("Enter Gitlab API access token:")
+
+        fetch_binaries.fetch_build_artifacts(
+            args.try_fetch_from_pipeline,
+            target_os="uniffi",
+            target_arch=None,
+            token=token,
+            download_dir=PROJECT_ROOT,
+        )
+    else:
+        rutils.generate_uniffi_bindings(
+            PROJECT_CONFIG,
+            LIBTELIO_ENV_UNIFFI_GENERATORS_TAG,
+            ["python", "cs", "go", "swift", "kotlin"],
+            "src/libtelio.udl",
+            dockerized=args.dockerized,
+        )
+
+
 def exec_build(args):
+    # Try fetching from pipeline, if it fails, continue with the build
+    if args.try_fetch_from_pipeline:
+        if "LLH_GROUP_TOKEN_FLAKY_TESTS" in os.environ:
+            token = os.environ["LLH_GROUP_TOKEN_FLAKY_TESTS"]
+        else:
+            token = getpass.getpass("Enter Gitlab API access token:")
+        if fetch_binaries.fetch_build_artifacts(
+            args.try_fetch_from_pipeline,
+            args.os,
+            args.arch,
+            token,
+            PROJECT_ROOT,
+        ):
+            if args.moose and args.os in ["linux", "windows", "android"]:
+                moose_utils.fetch_moose_dependencies(args.os, MOOSE_MAP[args.arch])
+
+            return 0
```

Comment:
My intention was to _try_ to fetch and if it fails (artifacts deleted, no nightly tag to date) than proceed with the ordinary build.

We need them because of the `finalize_win.py` script artifacts are not included, so need to re-download those leftovers.

Reply:
So maybe we can warn the user and even ask if he wants to proceed with the build?

Reply:
It will be stuck on CI than, if we'll wait for something from `stdin`. For the error handling, I can wrap this into `try{}except{}` maybe ...

Reply:
Fixed